blackbox/docs/admin/system-information.rst

```diff
@@ -1494,15 +1494,23 @@ should pass.
    with future versions of CrateDB. For this reason, you should create a new
    snapshot for each of your tables. (See :ref:`snapshot-restore`.)
 
-License expiry check
-....................
-
-This check warns you when your license is close to expiration. It will yield a
-``MEDIUM`` alert when your license is valid for less than 15 days and a
-``HIGH`` alert when your license is valid for less than a day.
+License check
+.............
+
+This check warns you when your license is close to expiration, is already
+expired, or if the cluster contains more nodes than allowed by your license. It
+will yield a ``MEDIUM`` alert when your license is valid for less than 15 days
+and a ``HIGH`` alert when your license is valid for less than a day.
+All other cases, like `already expired` or `max-nodes-violation`, it will result
+in a ``HIGH`` alert.
 It's highly recommended you request a new license when this check triggers in
```

Comment:
"We recommend that you request..."

(sorry for not catching this on the first pass!)